tests/test_reactive_static_policy.R

```diff
@@ -0,0 +1,69 @@
+library(dplyr)
+library(testthat)
+
+source(here::here("src/data_processing.R"))
+source(here::here("src/auto_scaling_algorithm.R"))
+
+# Load data files
+data <- processing_data("data/util_data.csv")
+data_decreasing <- read.csv("data/decreasing_data.csv")
```

Comment:
again. I think it is better to have test data below the tests directory

Reply:
Done